pkg/services/ngalert/state/template/error_test.go

```diff
@@ -0,0 +1,13 @@
+package template
```

Comment:
nit: I'm not sure I see the value of this test - you already test this incidentally on other functions.

Reply:
Those tests are in another package though which is the issue. If that package changes its behaviour then the implicit tests for `ExpandError` might no longer exist and then the code is untested.